en/release_note/release_note_latest_ver.rst

```diff
@@ -22,62 +24,66 @@ For CUBRID 9.3, please find https://www.cubrid.org/manual/en/9.3.0/release_note/
 Overview
 ========
 
-CUBRID 10.2 is the latest stable version that includes new features, significant changes and enhancements.
+CUBRID 11.0 is the latest stable version that includes new features, significant changes and enhancements.
 
 .. TODO: UPDATE WITH DETAILS.
 
-CUBRID 10.2
+CUBRID 11.0
 
-* is a significantly improved version.
-* is more reliable, faster and scalable: Online parallel data loading, optimizes auto-commit workloads
+* is a version with improved security.
+* is more stable, faster, and more convenient for administrators.
 * fixes a large number of critical bugs.
-* includes useful SQL extensions: JSON data type and CTE(Common Table Expressions) extensions.
-* includes many enhancements to index management: Online parallel index loading, Faster foreign key loading, Invisible Index.
-* includes huge scale of code refactoring and modernization.
-
-CUBRID 10.2 is **faster**. It optimizes auto-commit workloads, for instance, YCSB Workload A (140%: 10.1) and YCSB Workload B (130%: 10.1). For faster bulk data loading, CUBRID 10.2 has online parallel loader which shows x10 performance to CUBRID 10.1.
+* includes useful SQL extensions: Supporting RVC (Row Value Constructor) and various REGEXP functions.
+* includes code refactoring and modernization.
 
-CUBRID 10.2 is also **better**, bringing new features. CUBRID 10.2 supports built-in JSON data types, path, functions and operators. Common Table Expression is now allowed for INSERT, REPLACE, DELETE, UPDATE, CREATE TABLE AS SELECT as well as SELECT. \
-Index management is greatly improved. To load a secondary key is no more painful. It supports online index loading which allows other select and DMLs without loosing throughputs. It even provides parallel option to load an online index, which greatly speeds up index loading. \
-It also provides from x2 to x3 performance boost to load foreign keys. \
-Invisible index helps DBAs to test effect of removing an index without making a disruptive change.
+CUBRID 11.0 **improves security** by providing data encryption and packet encryption. This version prevents abnormal data leakage by supporting table-based TDE (Transparent Data Encryption) and packet encryption between the driver and server.
```

Comment:
@charischau 
How about changing "leakage" to "loss" ?

Reply:
okay sounds good to me :)

Reply:
I fixed it.